build/commands/lib/config.js

```diff
@@ -86,6 +86,7 @@ const Config = function () {
   this.targetOS = getNPMConfig(['target_os'])
   this.gypTargetArch = 'x64'
   this.targetApkBase ='classic'
+  this.targetOutput = 'apk'
```

Comment:
shouldn't the default for release be `aab`? specifying `Release` should set everything correctly and the only "overrides" we should have in CI are for things we can't set here like api keys and urls

Reply:
also we have two defaults here because of https://github.com/brave/brave-core/pull/8421/files#diff-c762921b1126e4475bffab885d122da3939e98234baa81d784b613ee352bc9cdR61 so this one should be removed

Reply:
ex ```Config.prototype.isOfficialBuild = function () {
  return this.buildConfig === 'Release'
}```

Reply:
The duplicate is removed.